cpp/src/groupby/sort/group_std.cu

```diff
@@ -33,30 +35,56 @@ namespace cudf {
 namespace groupby {
 namespace detail {
 namespace {
-template <typename ResultType, typename T>
+
+template <typename ResultType, typename Iterator>
 struct var_transform {
-  column_device_view d_values;
-  column_device_view d_means;
-  column_device_view d_group_sizes;
+  column_device_view const d_values;
+  Iterator values_iter;
+  ResultType const* d_means;
+  size_type const* d_group_sizes;
   size_type const* d_group_labels;
   size_type ddof;
 
   __device__ ResultType operator()(size_type i)
   {
     if (d_values.is_null(i)) return 0.0;
 
-    ResultType x         = d_values.element<T>(i);
+    ResultType x = static_cast<ResultType>(values_iter[i]);
+
     size_type group_idx  = d_group_labels[i];
-    size_type group_size = d_group_sizes.element<size_type>(group_idx);
+    size_type group_size = d_group_sizes[group_idx];
 
     // prevent divide by zero error
     if (group_size == 0 or group_size - ddof <= 0) return 0.0;
 
-    ResultType mean = d_means.element<ResultType>(group_idx);
+    ResultType mean = d_means[group_idx];
```

Comment:
This is sort of the reverse direction of what I've been doing. `column_device_view`'s `element()` accessor is more helpful than simple subscript. e.g. in fixed point. `element()` will return the value with the scale applied. And that scale is stored once in the column.

Any particular reason for this change?